cocos/3d/skeletal-animation/skeletal-animation-state.ts

```diff
@@ -83,19 +83,13 @@ export class SkeletalAnimationState extends AnimationState {
         this._frames = frames - 1;
         this._animInfo = this._animInfoMgr.getData(root.uuid);
         this._bakedDuration = this._frames / samples; // last key
+        this.setUseBaked(baked);
```

Comment:
This ensure the bake option of animation state is up-to-date with skinned mesh renderer, whenever.